test/test_latlng.rb

```diff
@@ -254,4 +254,24 @@ def test_to_dms
     assert_equal [-87, 39], dms[0, 2]
     assert_equal 37, dms[2].round
   end
+
+  def test_invalid_or_unsupported_units
+    exception = assert_raise(ArgumentError) do
+      @loc_a.distance_to(@loc_e, units: 'invalid_units')
+    end
+
+    assert_equal(
+      'invalid_units is an unsupported unit of length.',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.